test/jdk/java/util/zip/DeflaterClose.java

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```
Same for `InflaterClose.java`.

Reply:
Hello Chen, these tests were actually introduced in a RFR back in 2019, so the original year is correct. I've now converted these tests to junit and updated the PR and updated the copyright years as well.